ansible_collections/arista/avd/tests/unit/roles/eos_validate_state/tests/test_avdtestrouting.py

```diff
@@ -222,5 +222,25 @@
         "expected_log": "Peer group 'IPv4-UNDERLAY-PEERS' from the 'peer_groups' list of the 'router_bgp' data model is missing the variable 'type'.",
         "expected_log_level": "WARNING",
     },
+    {
+        "test_name": "missing-the-only-neighbor-ip-address",
+        "test_module": "AvdTestBGP",
```

Comment:
Looks like it should be in another PR ? I am fine to leave it in there if it is an additional test for something already implemented though.